src/api/app/controllers/trigger_workflow_controller.rb

```diff
@@ -70,21 +63,15 @@ def set_workflow_run
                                              workflow_configuration_url: @token.workflow_configuration_url,
                                              scm_vendor: scm_vendor,
                                              hook_event: hook_event)
-  end
-
-  # There are plenty of SCM events we do not handle at all (yet).
-  # Instead of throwing and errror we just ignore them.
-  def skip_unsupported_events
-    return if @workflow_run.supported_event?
+    @workflow_run.valid?
+    return unless @workflow_run.errors.any?
 
-    render_ok(data: { info: "Ignored unsupported event: '#{@workflow_run.hook_event}'" })
-  end
-
-  # There are plenty of SCM event actions we do not handle at all (yet).
-  # Instead of throwing and errror we just ignore them.
-  def skip_unsupported_actions
-    return if @workflow_run.supported_action?
-
-    render_ok(data: { info: "Ignored unsupported '#{@workflow_run.hook_event}' event action: '#{@workflow_run.hook_action}'" })
+    # There are plenty of SCM events we do not handle at all (yet).
+    # Instead of throwing and errror we just ignore them.
+    if @workflow_run.errors[:hook_action].any? || @workflow_run.errors[:hook_event].any?
```

Comment:
So other errors are ignored? Why so specific?

Reply:
it was already like this, we used to check if the webhook is for a supported action/event. If that was not the case we still gave back a `200` with an info that we don't support those events. If I remember correctly we did so, because people did forget to uncheck some unsupported events in the webhook configuration and the list of bad request was growing in the list of webhook deliveries on the SCM side. They thought its broken, because the webhook deliveries that actually succeeded were buried under the failing once. But since the validation for the correct event and action is now moved to the workflow_run model, I had to check for those two error cases and only then deliver a 200. In all other cases its still a 400 with the error message (two lines below)

Reply:
we did this in `skip_unsupported_actions`

Reply:
Hm okay, but you will never get here because of the return above?